tracking/src/main/java/org/hps/recon/tracking/kalman/KalTrack.java

```diff
@@ -514,6 +515,49 @@ public Pair<Double[], Double> unbiasedIntersect(int layer, boolean local) {
         }
     }
 
+    public Pair<Double,Double> getIsoAndT0(int layer){
+         if (lyrMap == null) {
+            makeLyrMap();
+         }
+         if (lyrMap.containsKey(layer)) {
+             return getIsoAndT0(lyrMap.get(layer));
+         }else{
+             return new Pair<Double, Double>(-999., -999.);
+         }         
+    }
+
+    public Pair<Double, Double> getIsoAndT0(MeasurementSite ms){
+        double iso=-999.;
+        double isot0=-999.;
+        SiModule sensor=ms.m; 
+        List<Measurement> allHits=sensor.hits;
+        if(allHits.size()>2 && ms.hitID>-1){
```

Comment:
why is this >2? is it because you need at least two hits to form a (strip?)cluster for that sensor?